services/reuse/reuse-compliance.service.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+const Joi = require('joi')
+const { BaseJsonService } = require('..')
+const { renderReuseBadge } = require('./reuse-compliance-helper')
+
+const responseSchema = Joi.object({
+  badge: Joi.string(),
+  hash: Joi.string(),
+  last_access: Joi.date(),
+  lint_code: Joi.number(),
+  lint_output: Joi.string(),
+  status: Joi.string().required(),
+  url: Joi.string().required(),
+}).required()
+
+module.exports = class Reuse extends BaseJsonService {
+  static category = 'license'
+
+  static route = {
+    base: 'reuse/compliance',
+    pattern: ':gitserver/:user/:repository',
+  }
+
+  static examples = [
+    {
+      title: 'REUSE Compliance',
+      namedParams: {
+        gitserver: 'github',
+        user: 'fsfe',
+        repository: 'reuse-tool',
+      },
+      staticPreview: this.render({ status: 'compliant' }),
+      keywords: ['license'],
+    },
+  ]
+
+  static defaultBadgeData = {
+    label: 'REUSE',
```

Comment:
Label should be lower-case. See https://github.com/badges/shields/blob/master/CONTRIBUTING.md#badge-guidelines